provision/acc_provision/apic_provision.py

```diff
@@ -942,6 +944,8 @@ def kube_dom(self):
         elif vmm_type == "CloudFoundry":
             mode = "cf"
             scope = "cloudfoundry"
+        elif apic_version >= 5.1 and cluster_provider == "Rancher":
```

Comment:
Why do we reply on apic_version to set the mode? Can we not just call it out explicitly?

Reply:
rancher 'mode' was added in 5.1. so the post will fail with earlier versions. Rather than rejecting rancher usage with older ACI versions we set the mode to k8s as the mode is only used by the UI and rancher/k8s are identical in the way they are handled